src/connection/store.ts

```diff
@@ -115,7 +115,7 @@ export class ConnectionStore {
                     accountantId,
                     serviceType: this.root.proposals.active.serviceType,
                 },
-                5000,
+                30000,
```

Comment:
@anjmao have you experienced anything like this: https://github.com/mysteriumnetwork/node/issues/1996 ?